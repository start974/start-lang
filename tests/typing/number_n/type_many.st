def a : N := 1
def b := 2
def a := 3
def c := 5
