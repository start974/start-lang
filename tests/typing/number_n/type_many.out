def a : N := 1
def b : N := 2
def a : N := 3
def c : N := 5
